scripts/mkimage

```diff
@@ -75,7 +76,7 @@ trap cleanup SIGINT
 # create part1
   echo "image: creating part1..."
   SYSTEM_PART_END=$(( $SYSTEM_SIZE * 1024 * 1024 / 512 + 2048 ))
```

Comment:
No, I'm pretty sure this is the offset, so 2048 needs to be replaced with `$SYSTEM_PART_START`. The 1MB "spacer" is added to the starting sector of the second partition. If you use a larger offset than 2048 you'll effectively reduce the the size of the first partition by the difference between the new offset and 2048 (calculated in sectors).